src/main/java/teammates/ui/automated/PutCommentDocumentWorkerAction.java

```diff
@@ -0,0 +1,30 @@
+package teammates.ui.automated;
+
+import teammates.common.datatransfer.attributes.CommentAttributes;
+import teammates.common.util.Assumption;
+import teammates.common.util.Const;
+
+/**
+ * Task queue worker action: Puts searchable document for comment.
+ */
+public class PutCommentDocumentWorkerAction extends AutomatedAction {
+
+    @Override
+    protected String getActionDescription() {
+        return null;
+    }
+
+    @Override
+    protected String getActionMessage() {
+        return null;
+    }
+
+    @Override
+    public void execute() {
+        String commentId = getRequestParamValue(Const.ParamsNames.COMMENT_ID);
+        Assumption.assertNotNull(commentId);
+
+        CommentAttributes comment = logic.getComment(Long.valueOf(commentId));
```

Comment:
I think it might be possible due to eventual consistency (when the comment hasn't persisted to all database servers yet), but okay, we probably aren't ready to handle such cases yet. Or it could happen if the comment was added then immediately deleted, before the task ran. Could you change it to a check for null and if it is, log severe and return?